### PR TITLE
Avoid naming collisions

### PR DESCRIPTION
**Pull request by gubatron**
Fixes possible bugs due to naming collisions with built-in types and imported package names.

### CODE REVIEW

File: addrmgr/network.go [hunk 1]
```diff
@@ -259,8 +259,8 @@ func GroupKey(na *wire.NetAddress) string {
 		// teredo tunnels have the last 4 bytes as the v4 address XOR
 		// 0xff.
 		ip := net.IP(make([]byte, 4))
-		for i, byte := range na.IP[12:16] {
-			ip[i] = byte ^ 0xff
+		for i, byte_ := range na.IP[12:16] {
```
Comment (tyler-smith) [inline addrmgr/network.go]: Really small nitpick but generally Go identifiers to use underscores. What do you think about `ipByte` or `addressByte`?

Comment (cpacia) [reply addrmgr/network.go]: Yeah it will need to be changed to pass the linter

File: cmd/bchctl/config.go [hunk 2]
```diff
@@ -68,9 +68,9 @@ func listCommands() {
 			continue
 		}
 
-		// Categorize the command based on the usage flags.
+		// Categorize the command based on the usage usageFlags.
```
Comment (cpacia) [inline cmd/bchctl/config.go]: Also i'd just update this comment to say `// Categorize the command based on the usageFlags.`

**Comment by cpacia** [comment]
Looks good. Thanks @gubatron!

**Pull request merged**